ONSdigital/dp-frontend-cookie-controller
# Feature/middleware handler

### What

Implemented new controller middleware that should guarantee the language always has a default. Values are no longer set via the headers and into the receiving end. Middleware will instead extract values that are to be passed into the handlers.

### How to review

- Check that the middleware has been implemented correctly
- Check that the `lang` value is set in all. At present the default locale in dp-net is set to "en" this should be the value found if set.
- The DOM should state: `<html xml:lang="en" lang="en"><head>`

### Who can review

Anyone
